sympy/concrete/expr_with_limits.py

```diff
@@ -592,7 +592,8 @@ def _eval_factor(self, **hints):
 
     def _eval_expand_basic(self, **hints):
         summand = self.function.expand(**hints)
-        if summand.is_Add and summand.is_commutative:
+        if (summand.is_Add and summand.is_commutative and
+                self.has_finite_limits is not False):
```

Comment:
Maybe it would be good to check `hints['force']` then someone could force the expansion by passing `force=True`.

Reply:
That's a good idea.  I'll create another PR for this and update the .mailmap